dpub/ref.py

```diff
@@ -32,6 +32,24 @@ def next_cell(cell, majorDimension=drive.ROWS_DIMENSION):
     return join_cell(letter, number)
 
 
+def cell_for_new_test(cell, tests_map_len, majorDimension=drive.ROWS_DIMENSION):
```

Comment:
Let's add some unitary tests for simple methods like this one to prevent regressions.

Reply:
It's pending to be done yet.